rest/rest-api/src/main/java/org/apache/brooklyn/rest/api/EntityApi.java

```diff
@@ -141,7 +141,7 @@ public Response getIcon(
 
     @GET
     @Path("/{entity}/tags")
-    @ApiOperation(value = "Fetch list of tags on this entity")
+    @ApiOperation(value = "Fetch list of tags on this entity. For retrieving brooklyn.tags filter only elements such that: an element is a map AND an element has kind: camp_brooklyn_tags.")
```

Comment:
Not sure we want to include this in the rest call's description. It's very technical, and doesn't tell people how to use the information (e.g. "filter" is not something that the rest api supports, it has to be done by the user after receiving the entire response).

I think this would be better to just include in the docs, with a more complete example of how to do it.